maven-plugin-tools-java/src/main/java/org/apache/maven/tools/plugin/extractor/javadoc/JavaJavadocMojoDescriptorExtractor.java

```diff
@@ -56,11 +56,13 @@
 
 /**
  * <p>
```

Comment:
Do we need to convert javadoc -> XHTML for the legacy extractor as well?

Reply:
I would not touch legacy extractors